source/safe-cast-to.ts

```diff
@@ -0,0 +1,27 @@
+/**
+Cast a value to the given type safely.
+
```

Comment:
It would be useful with a short explanation on how it's "safe" and better than just `as Partial<Foo>`.